src/common/pageLayout/PageLayout.tsx

```diff
@@ -8,9 +10,24 @@ import styles from './PageLayout.module.css';
 type Props = React.PropsWithChildren<{
   className?: string;
   hideFooterLogo?: boolean;
+  title?: string;
 }>;
 
 function PageLayout(props: Props) {
+  const { trackPageView } = useMatomo();
+  const { t } = useTranslation();
+  const { title = 'appName' } = props;
+
+  const pageTitle =
+    props.title !== 'appName' ? `${t(title)} - ${t('appName')}` : t('appName');
+
+  useEffect(() => {
+    trackPageView({
+      documentTitle: pageTitle,
+      href: window.location.href,
+    });
```

Comment:
This `useEffect` has the same issue I mentioned in the previous PR. I'm not entirely sure, but I think that you had trouble with the linter then so I am highlighting again in case that's the case here as well.

Reply:
Nice catch. Seems i'm relying a little too much on linting 😅